internal/cmd/users.go

```diff
@@ -346,14 +330,19 @@ func updateUser(cli *CLI, name string) error {
 	return nil
 }
 
+// Throws error if none or multiple results are found
 func getUserByName(client *api.Client, name string) (*api.User, error) {
 	users, err := client.ListUsers(api.ListUsersRequest{Name: name})
 	if err != nil {
+		if api.ErrorStatusCode(err) == 403 {
+			logging.Debugf("%s", err.Error())
```

Comment:
Could we centralize this error debug logging somewhere?

Reply:
Do you have any ideas? I was wondering about this too. Each client call /  response  to be logged everywhere, eg) 
instead of cli logging `call server: list users`, it would seem nicer if it was logged beside the call itself (eg. in the api/client.go itself?)

Reply:
I think that is a good idea. Looks like there is already a `checkError` function in `client.go`. The debug log could fit in there nicely.